diff --git a/code/common.mk b/code/common.mk
index 3e6cd46..5736124 100644
--- a/code/common.mk
+++ b/code/common.mk
@@ -1,10 +1,10 @@
-CROSS_COMPILE = riscv64-unknown-elf-
+CROSS_COMPILE = riscv64-elf-
 CFLAGS = -nostdlib -fno-builtin -march=rv32g -mabi=ilp32 -g -Wall
 
 QEMU = qemu-system-riscv32
 QFLAGS = -nographic -smp 1 -machine virt -bios none
 
-GDB = gdb-multiarch
+GDB = riscv64-elf-gdb
 CC = ${CROSS_COMPILE}gcc
 OBJCOPY = ${CROSS_COMPILE}objcopy
 OBJDUMP = ${CROSS_COMPILE}objdump
\ No newline at end of file
